asmcomp/selectgen.ml

```diff
@@ -706,6 +707,13 @@ method emit_expr (env:environment) exp =
           dbg, Cconst_int (1, dbg),
           dbg, Cconst_int (0, dbg),
           dbg))
+  | Cop(Copaque, args, dbg) ->
+      begin match self#emit_parts_list env args with
+        None -> None
+      | Some (simple_args, env) ->
+         let rs = self#emit_tuple env simple_args in
+         Some (self#insert_op_debug env Iopaque dbg rs rs)
+      end
```

Comment:
ah, someday we will be able to write something like `Option.let+ simple_args, env = self#emit_parts_list env args in ...`.